### ISSUE TITLE
Feature Request: User must be able to click on product title at products page  and go to product details page .

### ISSUE THREAD

**Issue by Devika-Sahu** [varunKT001/tomper-wear-ecommerce#47]
### Is your feature request related to a problem? Please describe.

It is frustrating for a user when he is not able to access the product details page to confirm the details of the products present in his cart. Users must be given  the functionality to access product details from the cart page.

<img width="960" alt="image" src="https://user-images.githubusercontent.com/75083536/159030271-87cf285c-103f-4c08-97d6-a2eeb0fc15a2.png">

Circled part should be clickable. 

### Describe the solution you'd like.

User must be able to click on product title at products page  and go to product details page . It will smoothen the shopping experience for users

### Additional context (if any)

Please assign this task to me.

**Comment by Siddhesh777** [comment]
I can work on this @varunKT001 .
If possible please assign this issue to me.

**Comment by varunKT001** [comment]
Since @Devika-Sahu has created this issue, I can't assign this to you 😕

She is working on another issue that's why this is one is unassigned 👍

**Comment by Devika-Sahu** [comment]
@varunKT001 I have completed the previous task. Can I start working on this now?

**Comment by varunKT001** [comment]
Wait for it to merge, I'll assign you this issue once your pull request is merged 👍
